types/steam-user/index.d.ts

```diff
@@ -859,7 +867,33 @@ interface Events {
 //#endregion "Events"
 
 //#region "Helper Types"
-type RegionCode = 0x00 | 0x01| 0x02 | 0x03 | 0x04 | 0x05 | 0x06 | 0x07 | 0xFF; // https://developer.valvesoftware.com/wiki/Master_Server_Query_Protocol#Region_codes
+type RegionCode = 0x00 | 0x01 | 0x02 | 0x03 | 0x04 | 0x05 | 0x06 | 0x07 | 0xFF; // https://developer.valvesoftware.com/wiki/Master_Server_Query_Protocol#Region_codes
+type OwnsFilterFunction = (element: Proto_CMsgClientLicenseList_License, index: number, array: Proto_CMsgClientLicenseList_License[]) => boolean;
+interface Proto_CMsgClientLicenseList_License {
+    package_id: number;
+    time_created: number;
+    time_next_process: number;
+    minute_limit: number;
+    minutes_used: number;
+    payment_method: SteamUser.EPaymentMethod;
+    flags: number;
```

Comment:
FYI I just pushed a jsdoc change to properly refer to `flags` and `license_type` with their enum types: https://github.com/DoctorMcKay/node-steam-user/commit/9312326c34fed69a2ea6c4102ed25e3073f0516d

I'm not going to bother publishing such a minor doc change to npm, but you might want to go ahead and change these here.

Reply:
Thanks for letting me know!